atlp-rwanda/Team-Sostene-E-commerce-bn
# #184651081 Buyer should be able to track their order status and get notified to the frontend.

#### What does this PR do?
It handles the retrieval and updating of order status information, and provide this information to the frontend for display.
#### Description of Task to be completed?
The appropriate order status information should be retrieved from the database
The order status information should be sent to the frontend to be displayed to the buyer
The order status in the database should be updated
The updated order status should be sent to the frontend to be displayed to the buyer
#### How should this be manually tested?
Clone the repository and npm install to install all dependencies
Seng a GET request to retrieve the order's status via /orders/id
The order's status will be returned as a response and notification should be sent to the frontend
#### Any background context you want to provide?

#### What are the relevant pivotal tracker/Trello stories?
https://www.pivotaltracker.com/story/show/184651081
#### Screenshots (if appropriate)

#### Questions:
